audio.py

```diff
@@ -88,3 +137,20 @@ def compute_duration(audio_path, backend = 'ffmpeg'):
 	cmd = ['ffprobe', '-v', 'error', '-show_entries', 'format=duration', '-of', 'default=noprint_wrappers=1:nokey=1'
 			] if backend == 'ffmpeg' else ['soxi', '-D'] if backend == 'sox' else None
 	return float(subprocess.check_output(cmd + [audio_path]))
+
+
+if __name__ == '__main__':
+	parser = argparse.ArgumentParser()
+	parser.add_argument('--audio_path', type = str, required = True)
+	parser.add_argument('--sample-rate', type = int, default = 8000, required = True)
+	parser.add_argument('--mono', type = bool, action = 'store_true')
+	parser.add_argument('--audio-backend', type = str, required = True)
+	parser.add_argument('--show-time', type = str, action = 'store_true')
+	parser.add_argument('--n-times', type = int, default = 100)
+	args = parser.parse_args()
+
+	start = time.time()
+	for i in range(args.n_times):
+		signal, sample_rate = read_audio(args.audio_path, sample_rate=args.sample_rate, mono=args.mono, backend=args.audio_backend, show_time=args.show_time)
+	print(f'Sum time per {args.n_times} times: ', time.time() - start)
```

Comment:
Please record the times, and only then do prints. Otherwise, print can slightly skew the results (it may have to wait for IO, and some counters may include this time).

Reply:
done